src/components/sidebar-image-grid/SidebarImageGrid.tsx

```diff
@@ -31,28 +34,37 @@ const SidebarImageGrid: FC<SidebarImageGridProps> = ({
           icon={<ImageOutlinedIcon />}
           title={image.name ?? t(`chatPage.sidebar.unknownName`)}
         >
-          <Box sx={styles.imageWrapper}>
-            <Box component='img' src={image.path} sx={styles.modalImage} />
-          </Box>
+          <Box component='img' src={image.path} sx={styles.modalImage} />
         </AllContentModal>
       )
     })
   }
 
-  const compactGrid = images.slice(maxElemToShow * -1).map((image, index) => (
-    <ClickableImage image={image} key={image._id} onClick={showImage}>
-      {index === 2 && <Box>+{mediaSize - 2}</Box>}
-    </ClickableImage>
-  ))
+  const compactGrid = images.slice(0, maxElemToShow).map((image, index) => {
+    const hasMoreElem = index === maxElemToShow - 1 && mediaSize > maxElemToShow
+
+    return (
+      <ClickableImage
+        image={image}
+        key={image._id}
+        onClick={hasMoreElem ? onClick : showImage}
+      >
+        {hasMoreElem && (
+          <Box>
```

Comment:
Would it be more suitable to use an empty fragment instead of a Box?

Reply:
inside ClickableImage component it sets some styles to '& div', so it should be wrapped by Box